### DISCUSSION TITLE
how to set automaticly desktop audio input?

### DISCUSSION THREAD

**Discussion by RonXTCdaBass** [rocksdanister/lively#1826] [Q&A] [answered]
Hello!

i wanna use this at an old win7 pc and there no config menu is working. 
how to set "desktop audio" in the json or js files to standard / automatic, like in the fluid background?

thanks!

**Comment by rocksdanister** [comment, marked as answer]
I don't know what `standard/automatic` mean here but Lively does not support Win7:
https://github.com/rocksdanister/lively/wiki/Getting-Started

**Comment by RonXTCdaBass** [comment]
Thx for the answer!

i use the Lively v0.9.6.0 version and the most is working.
"Lively_Fluids_v3" react to my audio and it´s customizeable 
the simple "audio visualizer" react to my audio and it´s customizable too...

only the "AudiOrbits" dont let me customize. 

the web version (https://orbits.hexx.one) accept my audio but the wallpaper version not.
so i´m asking how to change or set "desktop audio" in the files of the html & js powered desktop.
maybe a code snippet?

sorry, english is not my mothertongue.

![lively-customize-error](https://github.com/rocksdanister/lively/assets/115949021/40f98a81-ac81-4466-a3d4-7fc61c305b89)

the web version (https://orbits.hexx.one) accept my audio but the wallpaper version not.

i hope now someone can understand what i mean.

**Reply by rocksdanister** [reply]
I don't provide support for older versions, the error you are getting is because of outdated version.

The Lively API docs can be found here if you wish to figure things out:
https://github.com/rocksdanister/lively/wiki/Web-Guide-V-:-System-Data
